tracer/src/Datadog.Trace/AppSec/Waf/NativeBindings/WafNative.cs

```diff
@@ -488,25 +488,6 @@ private static bool TryLoadLibraryFromPaths(string libName, List<string> paths,
             return success;
         }
 
-        private static bool IsMuslBasedLinux()
```

Comment:
I think this is still needed. We have two different folder layouts for distributing binaries.

- tracer-home - contains binaries for a specific OS
- dd-tracer-home - contains binaries for all OSs

The tests are currently using tracer-home, but this will be removed, in favour of dd-tracer-home, where this function is needed to select the right binrary.

Reply:
as discussed off line this is no longer true since the waf doesn't have a musl specific version anymore.